lib/Targets/AIETargetXAIEV1.cpp

```diff
@@ -153,6 +152,7 @@ mlir::LogicalResult AIETranslateToXAIEV1(ModuleOp module, raw_ostream &output) {
       output << clear_range(col, row, 0x3F200, 0x3F3AC);
 
       // Release locks
+      int numLocks = target_model.getNumLocks(col, row);
       output << "for (int l=0; l<" << numLocks << "; l++)\n"
```

Comment:
While you are here, apply the coding style to the generated code
```suggestion
      int numLocks = target_model.getNumLocks(col, row);
      output << "for (int l = 0; l < " << numLocks << "; ++l)\n"
```

Reply:
strange that clang-format didn't catch it.

Reply:
Because it is hidden in a string. :-) Perhaps you need a kind ChatGPT-powered clang-format... ;-)

Reply:
Ah, right.